doc/tor.1.txt

```diff
@@ -3044,6 +3044,32 @@ The next section describes the per service options that can only be set
    The HAProxy version 1 protocol is described in detail at
    https://www.haproxy.org/download/1.8/doc/proxy-protocol.txt
 
+[[HiddenServiceExportRendPoint]] **HiddenServiceExportRendPoint** **0**|**1**::
+   When HiddenServiceExportCircuitID along with this feature is enabled,
+   export IP and port of the rendezvous point along with the circuit ID. +
+ +
+   When this feature is enabled, tor will export IP and port of the rendezvous
+   point in the following way. +
+ +
+   "PROXY TCP6 fc00::AAAA:BBBB:CCCC:0:ffff:ffff ::1 65535 42\r\n" +
+ +
+      ip = (0xAAAA << 16) + 0xBBBB; +
+      port = 0xCCCC; +
+ +
+   For example, if the IP is 0x7f000001 and the port is 0x1bb, you can
+   translate to 127.30.0.1:443. (Default: 0)
+
+[[HiddenServiceExportInstanceID]] **HiddenServiceExportInstanceID** __N__::
+   When HiddenServiceExportCircuitID is enabled, export an instance ID along
```

Comment:
Why require HiddenServiceExportCircuitID to be enabled?

Reply:
Instead, we could enable this feature when HiddenServiceExportInstanceID is non-zero.

Reply:
See https://github.com/torproject/tor/pull/1562#discussion_r353520163 .